Makefile

```diff
@@ -0,0 +1,47 @@
+TOP ?= $(shell git rev-parse --show-toplevel)
+
+export PARSECDIR := $(TOP)
+export PATH := $(TOP)/bin:$(PATH)
+export MANPATH := $(TOP)/man:$(MANPATH)
+INSTALL_DIR ?= $(TOP)/../../install
+LOG_DIR ?=  $(TOP)/../../outputs
+export EXT ?= riscv
+ifeq ($(EXT),x86)
+WORKING_BENCHMARKS = blackscholes bodytrack facesim ferret fluidanimate freqmine streamcluster swaptions vips canneal dedup raytrace
+export TOOLCHAIN_PREFIX=
+# check these paths, for most users this just needs to be "usr/"
+export CC_DIR=
+export UTIL_DIR=
+export GNUTOOL_DIR=
+# --------------
+export EMULATOR=
+export build_host=
```

Comment:
is there a sane default for these? what happens when they are left undefined?

Reply:
For the dirs they could be `/user`. The gcc.bldconf tool then adds a `/bin`, so for kk9 this matches the path for different tools located in `/bin/*`. For x86, no need for the qemu emulator, leaving it blank makes configuring the run targets easier later. Build host defaults to x86 when left blank.